phoenicis-tools/src/main/java/org/phoenicis/tools/lnk/AbstractLnkFlagsParser.java

```diff
@@ -0,0 +1,22 @@
+package org.phoenicis.tools.lnk;
+
+/**
+ * This class represent a byte flags.
+ * It helps to apply a mask and read them
```

Comment:
- where/on what is the mask applied?
- `and read them`: whom/what?

Reply:
On the flags. Honestly, I don't think we should explain everything or we're going to take time to discuss about binary operators.

Reply:
The issue is that the second sentence is kind of out of context.
You should extend it with a few more word so that it becomes more clear.
For example:
`It helps to apply a mask on the byte flags and allows reading them`